syft/workers/base.py

```diff
@@ -453,6 +455,26 @@ def send(
 
         return pointer
 
+    def garbage(self, object_id, location):
+        """
+        Garbage manager which collects all the remote GC request and batch send
+        them every "delay" seconds for every location.
+        """
+        max_delay = self.object_store.garbage_delay
+        max_size = self.object_store.trash_capacity
+        trash = self.object_store.trash
+
+        if location.id not in trash:
```

Comment:
PICKY: Make ```trash``` a ```defaultdict```.
```trash = defaultdict(lambda: (time.time(), []))```
Whenever you do not have a key and you want to access it, it will automatically create that tuple